easybuild/easyconfigs/f/FriBidi/FriBidi-1.0.2-GCCcore-6.4.0.eb

```diff
@@ -0,0 +1,34 @@
+# Authors:: Jack Perdue <j-perdue@tamu.edu> - TAMU HPRC - http://hprc.tamu.edu
+
+easyblock = 'ConfigureMake'
+
+name = 'FriBidi'
+version = '1.0.2'
+
+homepage = 'https://github.com/fribidi/fribidi'
+
+description = """
+ The Free Implementation of the Unicode Bidirectional Algorithm.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+
+# https://github.com/fribidi/fribidi/releases/download/v1.0.2/fribidi-1.0.2.tar.bz2
+source_urls = ['https://github.com/%(namelower)s/%(namelower)s/releases/download/v%(version)s']
+sources = [SOURCELOWER_TAR_BZ2]
+checksums = ['bd6d1b530c4f6066f42461200ed6a31f2db8db208570ea4ccaab2b935e88832b']
+
+builddependencies = [
+    ('Autotools', '20170619'),
+    ('binutils', '2.28'),
```

Comment:
@JackPerdue missing `'pkg-config` build dep? It's effectively used, see `configure` output

Reply:
fixed for you in https://github.com/JackPerdue/easybuild-easyconfigs/pull/2 ;)